macros/A11yRoleQuicklinks.ejs

```diff
@@ -1,25 +1,6 @@
 <%
 // Inserts quicklinks for the accessibility roles.
-var lang = env.locale;
-var url = "";
-
-switch(lang) {
-    case "de":
-        url = "/de/docs/Web/Accessibility/AT-APIs/Gecko/Rollen";
-        break;
-    case "fr":
-        url = "/fr/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
-        break;
-    case "ja":
-        url = "/ja/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
-        break;
-    case "ru":
-        url = "/ru/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
-        break;
-    default:
-        url = "/en-US/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
-        break;
-}
+var url = "/" + env.locale + "/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
```

Comment:
Your practice changes its old behavior. I think it should apply to all locales, shows the localized user interface and page.